src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -78,16 +79,16 @@ public CommandBase moveToDistance(double distance, double speed) {
         });
     }
 
-    public CommandBase movetoLow() {
-        return moveToDistance(Level.LOW.get(), SPEED);
+    public CommandBase moveToLow(Level LOW) {
```

Comment:
What. This completely defeats the purpose of parameterization

Reply:
Fixed it, I'll push later